.github/ISSUE_TEMPLATE/beta_feature.md

```diff
@@ -0,0 +1,59 @@
+---
+name: Beta Feature Requirements
+about: Requirements for promoting a feature to beta
+---
+
+This page lists the requirements for promoting a feature to beta. Please check off and document the steps as they are completed.
+
+**Feature Name:** 
+--- 
+
+### Requirements: 
+
+**Design**
+
+- [ ] Design doc has been approved
+
+**Config**
+
+- [ ] Can be enabled by default without requiring explicit user action. 
```

Comment:
This isn't really a checklist item, maybe the item is to consider whether the feature should be enabled by default?

Reply:
This was listed in the feature requirements from istio/community. However, I'm interpreting this as saying that in alpha, features must be disabled by default. In beta, they can either be enabled or disabled by default. As such, there is a restriction on alpha, but in beta anything goes.